src/Coduo/PHPMatcher/Matcher/Pattern/RegexConverter.php

```diff
@@ -19,6 +19,8 @@ public function toRegex(TypePattern $type)
                 return "(\\-?[0-9]*)";
             case 'double':
                 return "(\\-?[0-9]*[\\.|\\,][0-9]*)";
+            case 'null':
+                return "(null|NULL)";
```

Comment:
It's good, but with this change project was switched from 1.1.x to 1.1.4  - I think it was too big refactor to switch only the last number in version. It should be in 1.2 so when we've updated in project this library with 1.1.* format in our composer now we got an Exception.

Null is useless, thats true, so we will remove it from our mocks... but as I said - too big change for the last number :)